docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters/network-access.md

```diff
@@ -14,3 +16,17 @@ You can add IP addresses using any combination of the following options.
 | Create New IP Allow List | Create a new IP allow list and manually enter the CIDR and public IP addresses. |
 
 **Enable Public Access for this Cluster** - To connect to a cluster deployed in a VPC from a public IP address (including your current address), you must enable Public Access for the cluster. When enabled, a public IP address is added to each region of the cluster. You can view the private and public host addresses under **Connection Parameters** on the cluster **Settings > Infrastructure** tab.
+
+#### Private Service Endpoint
+
+For clusters in AWS and Azure, you can connect your cluster to your application VPC using a private link. To do this, you add a private service endpoint (PSE) to each region in your cluster.
+
+You can also add PSEs after your cluster is created.
```

Comment:
Provide a link to do it here : https://deploy-preview-19320--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/managed-endpoint-aws/#create-a-pse-in-yugabytedb-managed